Datahub.Core/Model/Datahub/PublicDataSharingRecords.cs

```diff
@@ -115,8 +115,10 @@ public class OpenDataSharedFile: SharedDataFile
     {
         public int? ApprovalForm_ID { get; set; }
         public string SignedApprovalForm_URL { get; set; }
-        public bool Read_FLAG { get; set; }
+        public bool ApprovalFormRead_FLAG { get; set; }
         public FileStorageType? FileStorage_CD { get; set; }
+        public OpenDataUploadStatus UploadStatus_CD { get; set; }
+        public string UploadError_TXT { get; set; }
```

Comment:
It looks like the migration is out of sync with the class. The migration has ApprovalFormRead and FileSize, but the class doesn't have FileSize and has UploadStatus and UploadError which are missing from the migration

Reply:
My mistake, I missed the other migration